### ISSUE TITLE
feature: add category css class to news article via parseArticles hook

### ISSUE THREAD

**Issue by fritzmg** [codefog/contao-news_categories#82]
It would be useful if the news entry's categories' CSS classes are added to the news article as well :)

**Comment by qzminski** [comment]
Meanwhile this is added by default you can use the following snippet in your ```news_``` templates:

```php
<?php
$categoryClasses = [];

foreach ($this->categories as $category) {
    $categoryClasses[] = $category['class'];
}
?>
<div class="layout_full block<?= $this->class ?> <?= implode(' ', $categoryClasses) ?>">
```

**Comment by qzminski** [comment]
Implemented in cb1a6c7.
